backend/src/main/game/cards/cards.py

```diff
@@ -0,0 +1,9 @@
+class Cards:
```

Comment:
Rename to something more obvious. It's going to be easy to confuse `Card` with `Cards`